templates/localgov-blogs-prev-next-block.html.twig

```diff
@@ -0,0 +1,27 @@
+{#
+/**
+* @file
+* Default theme implementation for the localgov_blogs_prev_next_block block.
+*/
+#}
+<nav class="localgov_blogs--navigation">
```

Comment:
Can we change this class to `localgov-blog-navigation`, what is there suggests this is a 'modifier' of `localgov_blogs` class (which doesn't actually exist any where).